_posts/2017-11-23-Rust-1.22.md

```diff
@@ -0,0 +1,166 @@
+---
+layout: post
+title: "Announcing Rust 1.22"
+author: The Rust Core Team
+---
+
+The Rust team is happy to announce the latest version of Rust, 1.22.0. Rust
+is a systems programming language focused on safety, speed, and concurrency.
+
+If you have a previous version of Rust installed, getting Rust 1.22 is as easy as:
+
+```bash
+$ rustup update stable
+```
+
+If you don't have it already, you can [get `rustup`][install] from the
+appropriate page on our website, and check out the [detailed release notes for
+1.22.0][notes] on GitHub.
+
+[install]: https://www.rust-lang.org/install.html
+[notes]: https://github.com/rust-lang/rust/blob/master/RELEASES.md#version-1220-2017-11-23
+
+## What's in 1.22.0 stable
+
+The headline feature for this release is one many have been waiting for
+for a long time: you can now [use `?` with `Option<T>`]! About a year
+ago, in [Rust 1.13], we introduced the `?` operator for working with
+`Result<T, E>`. Ever since then, there's been discussion about how far
+`?` should go: should it stay only bound to results? Should it be
+user-extensible? Should it be usable with `Option<T>`?
+
+In Rust 1.22, basic usage of `?` with `Option<T>` is now stable.
+This code will now compile:
+
+```rust
+fn try_option_some() -> Option<u8> {
+    let val = Some(1)?;
+    Some(val)
+}
+assert_eq!(try_option_some(), Some(1));
+
+fn try_option_none() -> Option<u8> {
+    let val = None?;
+    Some(val)
+}
+assert_eq!(try_option_none(), None);
+```
+
+However, this functionality is still a bit limited; you cannot yet write
+code that mixes results and options with `?` in the same function, for
+example. In the future, this will be possible, and already is in nightly
+Rust; expect to hear more about this in a future release.
+
+[use `?` with `Option<T>`]: https://github.com/rust-lang/rust/pull/42526
+[Rust 1.13]: https://blog.rust-lang.org/2016/11/10/Rust-1.13.html
+
+Types that implement `Drop` are [now allowed in `const` and `static`
+items](https://github.com/rust-lang/rust/pull/44456). Like this:
+
+```rust
+struct Foo {
+    a: u32
+}
+
+impl Drop for Foo {
+    fn drop(&mut self) {}
+}
+
+const F : Foo = Foo { a : 0 };
+static S : Foo = Foo { a : 0 };
+```
+
+On its own, this change doesn't bring much, but as we improve our
```

Comment:
How about `This change doesn't bring much on its own, but as we improve our...`?